lib/apis/fetch.js

```diff
@@ -8,9 +8,8 @@ export default (url, options = {}) => {
       method: 'GET',
       timeout: config.REQUEST_TIMEOUT_MS,
     });
-
     request(url, opts, (err, response) => {
-      if (!!err || response.statusCode !== 200) {
+      if (!!err || (response.statusCode !== 200 && response.statusCode !== 201)) {
```

Comment:
Yup! Even though our DSL here is called `fetch`, it just wraps `request` and let you pass in options such as `method: 'POST'`, so was just re-using rather than extracing/renaming.

Since this will thus be used for any GET/POST/PUT, any 200-level check make sense to me.